### PR TITLE
.Added logger for Action and  Press ESC key in multi drop down changes .

### PR DESCRIPTION
**Pull request by rahuldash171**

### CODE REVIEW

File: src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java [hunk 1]
```diff
@@ -1122,4 +1124,13 @@ public static void clickAddButtonAndEnterPropertyValues(String pluginProperty, L
                              listOfValues.get(valueIndex));
     }
   }
+
+  /**
+   * Escape after selecting a multi-select dropdown value
+   */
+  public static void pressEscapeKey() {
+    Actions act = new Actions(SeleniumDriver.getDriver());
```
Comment (poojantcs) [inline src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java:1133]: Start with logger. 
Suggested log text: "Press escape key"

File: src/main/java/io/cdap/e2e/utils/SeleniumDriver.java [hunk 2]
```diff
@@ -67,9 +67,9 @@ public class SeleniumDriver {
     url = executor.getAddressOfRemoteServer();
     waitDriver = new WebDriverWait(chromeDriver, ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
     chromeDriver.manage().timeouts()
-      .implicitlyWait(Duration.of(ConstantsUtil.IMPLICIT_TIMEOUT_SECONDS, ChronoUnit.SECONDS));
+        .implicitlyWait(Duration.of(ConstantsUtil.IMPLICIT_TIMEOUT_SECONDS, ChronoUnit.SECONDS));
```
Comment (poojantcs) [inline src/main/java/io/cdap/e2e/utils/SeleniumDriver.java]: Remove unnecessary changes

File: src/main/java/io/cdap/e2e/utils/SeleniumDriver.java [hunk 3]
```diff
@@ -67,9 +67,9 @@ public class SeleniumDriver {
...
     chromeDriver.manage().timeouts()
-      .pageLoadTimeout(Duration.of(ConstantsUtil.PAGE_LOAD_TIMEOUT_SECONDS, ChronoUnit.SECONDS));
+        .pageLoadTimeout(Duration.of(ConstantsUtil.PAGE_LOAD_TIMEOUT_SECONDS, ChronoUnit.SECONDS));
```
Comment (poojantcs) [inline src/main/java/io/cdap/e2e/utils/SeleniumDriver.java]: Remove unnecessary changes

File: src/main/java/stepsdesign/PipelineSteps.java [hunk 4]
```diff
@@ -742,4 +742,9 @@ public void enterKeyPropertyWithValue(String pluginProperty, DataTable table) {
     }
     CdfPluginPropertiesActions.clickAddButtonAndEnterPropertyValues(pluginProperty, keyValue);
   }
+
+  @Then("Press ESC key to close the dropdown")
```
Comment (poojantcs) [inline src/main/java/stepsdesign/PipelineSteps.java]: "Press ESC key"

File: src/main/java/stepsdesign/PipelineSteps.java [hunk 5]
```diff
@@ -742,4 +742,9 @@ public void enterKeyPropertyWithValue(String pluginProperty, DataTable table) {
...
+  public void pressESCKeyToCloseTheUniqueFieldsDropdown() {
```
Comment (poojantcs) [inline src/main/java/stepsdesign/PipelineSteps.java]: pressESCKey

**Review by itsankit-google — APPROVED** [review]
